templates/single.php

```diff
@@ -3,7 +3,7 @@
 <head>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1,user-scalable=no">
-	<link href="https://fonts.googleapis.com/css?family=Merriweather:400,400italic,700,700italic|Open+Sans:400,700,400italic,700italic" rel="stylesheet" type="text/css">
+	<link href="https://s1.wp.com/i/fonts/merriweather/merriweather.css?v=20160212" rel="stylesheet" type="text/css">
```

Comment:
Currently linked fonts can only be hosted at `fast.fonts.net` or `fonts.googleapis.com` so we'll have to use `@font-face` instead.

https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md#custom-fonts